src/Uno.UI/UI/Xaml/Style/mergedstyles.xaml

```diff
@@ -12687,7 +12687,7 @@
       </Setter.Value>
     </Setter>
   </android:Style>
-  <ios:Style TargetType="ProgressRing">
```

Comment:
This will be a breaking change. We'll likely need to enforce the native styles by default for this control on iOS/Android.

Reply:
Will do it inside of the `ProgressRing` code so I avoid messy platform specific XAML, maybe under with FeatureConfiguration to turn it off though

Reply:
Made native style default, and made docs to make it easy to override